manifests/plugin/gutterball/config.pp

```diff
@@ -0,0 +1,14 @@
+# gutterball config
+class katello::plugin::gutterball::config(
+    $foreman_user  = 'foreman',
+    $foreman_group = 'foreman',
+    $foreman_plugins_dir = '/etc/foreman/plugins',
```

Comment:
I still think this is the wrong directory - https://github.com/Katello/puppet-katello/pull/58#discussion-diff-23880066